modules/deployment-scripts/aks-run-command/README.md

```diff
@@ -17,6 +21,7 @@ An Azure CLI Deployment Script that allows you to run a command on a Kubernetes
 | `commands`                                 | `array`  | Yes      | An array of commands to run                                                                                   |
 | `initialScriptDelay`                       | `string` | No       | A delay before the script import operation starts. Primarily to allow Azure AAD Role Assignments to propagate |
 | `cleanupPreference`                        | `string` | No       | When the script resource is cleaned up                                                                        |
+| `scriptContent`                            | `string` | No       | Default Script to issue commands to AKS                                                                       |
```

Comment:
By allowing any script to be run in this module, (not just aks commands) doesn't that genericise it's capability a little too far?

What's the use case that this lights up?

Reply:
@Gordonby , if you check out the PR I have over here, https://github.com/Azure/azure-quickstart-templates/pull/12698, I have had to duplicate all the logic in this module. 
Generally, this lights up the ability to target AKS with Custom script actions.

In the current examples, there is one dedicated for HELM, but helm is not typically run on AKS, and instead targets AKS. This provides a more natural way to leverage helm with AKS.

(But I will give you its a bit of hack on what has been created)